src/lib/Behat/PageObject/SectionPage.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat\PageObject;
+
+
+use EzSystems\EzPlatformAdminUi\Behat\Helper\UtilityContext;
+use EzSystems\EzPlatformAdminUi\Behat\PageElement\AdminList;
+use EzSystems\EzPlatformAdminUi\Behat\PageElement\Dialog;
+use EzSystems\EzPlatformAdminUi\Behat\PageElement\ElementFactory;
+use EzSystems\EzPlatformAdminUi\Behat\PageElement\SimpleListTable;
+use EzSystems\EzPlatformAdminUi\Behat\PageElement\SimpleTable;
+use PHPUnit\Framework\Assert;
+
+class SectionPage extends Page
+{
+    /** @var string Name by which Page is recognised */
+    public const PAGE_NAME = 'Section';
+    /** @var string Name of actual group */
+    public $sectionName;
+
+    /** @var string locator for container of Content list */
+    public $secondListContainerLocator = 'section:nth-of-type(2)';
+
+    /**
+     * @var \EzSystems\EzPlatformAdminUi\Behat\PageElement\AdminList[]
+     */
+    public $adminLists;
+
+    /**
+     * @var \EzSystems\EzPlatformAdminUi\Behat\PageElement\AdminList
+     */
+    public $adminList;
+
+    /**
+     * @var \EzSystems\EzPlatformAdminUi\Behat\PageElement\Dialog[]
+     */
+    public $dialogs;
+
+    public function __construct(UtilityContext $context, string $sectionName)
+    {
+        parent::__construct($context);
+        $this->route = '/admin/section/view/';
+        $this->sectionName = $sectionName;
+        $this->adminLists['Section information'] = ElementFactory::createElement($this->context, AdminList::ELEMENT_NAME, 'Section information', SimpleTable::ELEMENT_NAME);
+        $this->adminLists['Content items'] = ElementFactory::createElement($this->context, AdminList::ELEMENT_NAME, 'Content items', SimpleTable::ELEMENT_NAME, $this->secondListContainerLocator);
+        $this->adminList = ElementFactory::createElement($this->context, AdminList::ELEMENT_NAME, '', SimpleTable::ELEMENT_NAME);
+        $this->dialogs['Section information'] = ElementFactory::createElement($this->context, Dialog::ELEMENT_NAME);
+        $this->pageTitle = sprintf('Section: %s', $sectionName);
+        $this->pageTitleLocator = '.ez-header h1';
+    }
+
+    /**
+     * Verifies that all necessary elements are visible.
+     */
+    public function verifyElements(): void
+    {
+        $this->adminLists['Section information']->verifyVisibility();
+        $this->adminLists['Content items']->verifyVisibility();
+    }
+
+    /**
+     * Verifies if list of Content items is empty.
+     *
+     * @param string $tabName
+     */
+    public function verifyListIsEmpty(): void
+    {
+        $firstRowValue = $this->adminLists['Content items']->table->getCellValue(1,1);
```

Comment:
I'd change it into two separate assertions:
Assert item count is equal to 1
Assert the message